src/CAREUI/misc/SectionNavigator.tsx

```
@@ -0,0 +1,78 @@
+        <Button
+          className={cn(
+            "justify-start bg-transparent shadow-none text-gray-700 hover:bg-black/5",
+            activeSection === section.id && "text-primary-500 font-semibold",
+          )}
+          onClick={() => {
+            const element = document.getElementById(section.id);
+            if (element) {
+              element.scrollIntoView({ behavior: "smooth" });
+            }
+          }}
+        >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)